plutus-ir/test/TransformSpec.hs

```diff
@@ -16,136 +15,8 @@ transform = testNested "transform" [
     ]
 
 thunkRecursions :: TestNested
-thunkRecursions = testNested "thunkRecursions" [
-    goldenPir "listFold" listFold,
-    goldenPir "monoMap" monoMap
+thunkRecursions = testNested "thunkRecursions"
+    $ map (goldenPir (runQuote . thunkRecursionsTerm) term)
```

Comment:
Perhaps missing something, but where is `term` bound?

Reply:
`goldenPir` takes as an argument the parser to be used for parsing the file, which is `term` (more properly, `Language.PlutusIR.Parser.term`) in this case (actually it's `term` in all the tests, so it might be ok to hard-code it into `goldenPir`, but it feels a bit...ugly)

Reply:
Ah, thanks.